ivy/functional/frontends/paddle/tensor/manipulation.py

```diff
@@ -132,3 +141,8 @@ def gather(params, indices, axis=-1, batch_dims=0, name=None):
 @to_ivy_arrays_and_back
 def roll(x, shifts, axis=None, name=None):
     return ivy.roll(x, shifts, axis=axis)
+
+
+@to_ivy_arrays_and_back
+def to_list(x):
+    return x.tolist()
```

Comment:
why do we need this extra function?

Reply:
It was pasted again at the end by mistake , I corrected it.

Reply:
I pushed again. @vaithak